### DISCUSSION TITLE
Backup

### DISCUSSION THREAD

**Discussion by dawidmachon** [ghostfolio/ghostfolio#1819] [Q&A]
I would like to ask how to proceed with backup and restoration process?

**Comment by dtslvr** [comment]
Hello @dawidmachon 

When it comes to backup and restoration processes, it depends on what kind of data you are trying to backup and restore. Within the app you can import and export your accounts and activities.

To create a full dump including historical market data, you can use a tool like `pg_dump`.

**Comment by swissbuechi** [comment]
You could run the backup script from [here](https://github.com/ghostfolio/ghostfolio/discussions/2786#discussioncomment-8658922) with a cronjob.
